lstchain/pointing/pointings.py

```diff
@@ -0,0 +1,86 @@
+import os
+import numpy as np
+from ctapipe.core import Component
+from ctapipe_io_lst.containers import LSTDriveContainer
+from ctapipe.core.traits import Unicode, Int
+
+__all__ = [
+    'PointingPosition'
+]
+
+
+class PointingPosition(Component):
+    """
+      Pointion position of telescopes
+
+    """
+    drive_path = Unicode(
+        '',
+        allow_none=True,
+        help='Path to the LST drive report file'
+    ).tag(config=True)
+
+    tel_id = Int(
+        0,
+        help='id of the telescope to take drive report for'
+    ).tag(config=True)
+
+    def _read_drive_report(self):
+        """
+        Reading drive reports
+
+        Returns:
+        data:`~astropy.table.Table`
+             A table of drive reports
+
+        """
+        from astropy.io import ascii
+        self.log.info("Drive report file:", self.drive_path)
+        if self.drive_path:
+                data = ascii.read(self.drive_path)
+            # Renaming the columns, since the drive report doesn't contain
+            # these information it self
+                data['col6'].name = 'time'
+                data['col8'].name = 'Az_avg'
+                data['col13'].name = 'El_avg'
+                return data
+        else:
+            raise Exception("No drive report file found")
+
+    def cal_pointingposition(self, ev_time):
+        """
+        Calculating pointing positions by interpolation
+
+        Parameters:
+        -----------
+        time: array
+            times from events
+
+        Returns
+        ----------
+        Azimuth: array
+
+        Elevation: array
+
+        """
+        timeL = ev_time[0]
+        timeH = ev_time[-1]
+        drive_container = LSTDriveContainer()
+        data = self._read_drive_report()
+        drive_container.time = data['time'].data
+        drive_container.Az_avg = data['Az_avg'].data
+        drive_container.El_avg = data['El_avg'].data
+        xp = drive_container.time
+
+        ind = np.where((xp >= timeL) & (xp <= timeH))
```

Comment:
In addition, the way to calculate the lower and upper elements of the drive time and their indices should be done like by adding these two lines:
```
lower_element = xp[xp < ev_time].max()
upper_element = xp[xp > ev_time].min()
```
and replacing` ind = np.where((xp >= timeL) & (xp <= timeH))`
by: 
```
ind = np.hstack(
                np.argwhere((xp >= lower_element) & (xp <= upper_element))
                )
```
The rest keeps the same

Reply:
Do you really need the indexes?
A mask will be faster
```
time_in_window = (xp >= timeL) & (xp <= timeH)
run_times = xp[time_in_window]
```

Reply:
Ok. We can modify accordingly.